File: app/src/main/java/com/snj/letschat/utils/CheckNet.java
package com.snj.letschat.utils;

import android.content.Context;
import android.net.ConnectivityManager;

public class CheckNet {

    public static Boolean isOnline(Context mContext) {

        ConnectivityManager cm = (ConnectivityManager) mContext
                .getSystemService(Context.CONNECTIVITY_SERVICE);

        return cm.getActiveNetworkInfo() != null
                && cm.getActiveNetworkInfo().isConnectedOrConnecting();
    }
}

File: app/src/main/java/com/snj/letschat/utils/CheckNet.kt
package com.snj.letschat.utils

import android.content.Context
import android.net.ConnectivityManager

class CheckNet{
     companion object {
         fun  isOnline(mContext: Context): Boolean {
             val cm = mContext
                     .getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
             return cm.activeNetworkInfo != null && cm.activeNetworkInfo.isConnectedOrConnecting
         }
     }

}
